src/renderer/uia/UiaRenderer.cpp

```diff
@@ -182,6 +182,20 @@ CATCH_RETURN();
     return S_FALSE;
 }
 
+[[nodiscard]] HRESULT UiaEngine::NotifyNewText(const std::wstring_view newText) noexcept
+try
+{
+    if (!newText.empty())
+    {
+        static constexpr std::wstring_view newline{ L"\n" };
+        _newOutput.reserve(_newOutput.size() + newText.size() + 1);
+        _newOutput.append(newText);
+        _newOutput.append(newline);

```

Comment:
IMO this is fine:
```cpp
_newOutput.append(newText);
_newOutput.push_back(L'\n');
```

Normally I'm a sucker for preallocating memory, but in this case the memory allocation will be amortized over time due to our `std::swap` trick. So in general, after a while, this here will practically never reallocate again.